fixtures/dom/public/react-loader.js

```diff
@@ -28,8 +28,8 @@ var query = parseQuery(window.location.search);
 var version = query.version || 'local';
 
 if (version !== 'local') {
-  REACT_PATH = 'https://unpkg.com/react@' + version + '/dist/react.min.js';
-  DOM_PATH = 'https://unpkg.com/react-dom@' + version + '/dist/react-dom.min.js';
+  REACT_PATH = 'https://unpkg.com/react@' + version + '/dist/react.js';
+  DOM_PATH = 'https://unpkg.com/react-dom@' + version + '/dist/react-dom.js';
```

Comment:
I switched to the unminified versions intentionally so was easier to step through stuff in devtools